tests/resources/test_common.py

```diff
@@ -122,20 +122,15 @@ def test_repr(self):
         Test __repr__
 
         """
-        patient1 = PatientItem(id="patient_id")
+        patient1 = PatientItem(id="patient_id1")
         self.assertEqual(
-            'PatientItem(\n\tid=patient_id,\n\tattributes='
-            '{\'id\': \'patient_id\'}\n)',
+            'PatientItem(id="patient_id1")',
             repr(patient1)
         )
 
-
-        patient2 = PatientItem(id="patient_id", key1="val1")
+        patient2 = PatientItem(id="patient_id2", key1="val1")
```

Comment:
How about also adding in a `name` field here and testing the `repr `with it.

Reply:
👍